src/test/java/uk/nhs/digital/nhsconnect/nhais/mapper/GpNameAndAddressMapperTest.java

```diff
@@ -16,8 +17,11 @@
     @Test
     void When_MappingGP_Then_ExpectCorrectResult() {
         Patient patient = new Patient();
+        Identifier identifier = new Identifier();
```

Comment:
Bartek made classes that wrap Identifier, see: `GeneralPractitionerIdentifier` and `ManagingOrganizationIdentifier`